_data/flux-commands.yaml

```diff
@@ -119,7 +127,42 @@ commands:
             command: flux alloc -N 2 --queue=foo
           - title: Create an allocation in the background with two nodes on queue 'foo'
             command: flux alloc --bg -N 2 --queue=foo
-            
+
+  - name: flux bulksubmit
+    groups:
+      - name: Format String Examples
+        items:
+          - title: Pipe inputs into flux bulksubmit
+            command: seq 1 4 | flux bulksubmit echo {}
+          - title: Map the same inputs into flux bulksubmit
+            command: "flux bulksubmit echo {} ::: 1 2 3 4"
+          - title: Remove the extension of the input string (e.g., one)
+            command: "flux bulksubmit echo {.%} ::: one.txt two.txt three.txt"
+          - title: Get the basename of the input string (e.g., path.txt)
+            command: "flux bulksubmit echo {./} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Combine the two - get the basename with extension removed (e.g., path)
+            command: "flux bulksubmit echo {./%} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Get the directory name of the input string (e.g., /long/file)
+            command: "flux bulksubmit echo {.//} ::: /long/file/path.txt /long/file/filename.txt"
+          - title: Get the input sequence number, start counting at 0
+            command: "flux bulksubmit echo {seq} ::: 1 2 3 4"
+          - title: Get the input sequence number, start counting at 1 (you monster)
+            command: "flux bulksubmit echo {seq1} ::: 1 2 3 4"
+          - title: Define a custom counter called p2
+            command: "flux bulksubmit --dry-run --define=p2='2**int(x)' -n {.p2} hostname \
+   ::: $(seq 0 4)"
+
+      - name: Simple Examples
+        items:
+          - title: Compress every file under ./mydir in parallel, one process per core
+            command: find ./mydir | flux bulksubmit --wait xz {} 
+          - title: Do a dry run and provide multiple inputs
+            command: "flux bulksubmit --dry-run /bin/bash -c 'touch mydir/{0}; echo touched number {1}' ::: filea fileb filec filed ::: 1 2 3 4"
```

Comment:
I'm not sure this examples works the way you might think it does. Just to double check, here's the result:
```console
$ flux bulksubmit --dry-run /bin/bash -c 'touch mydir/{0}; echo touched number {1}' ::: filea fileb filec filed ::: 1 2 3 4
bulksubmit: submit /bin/bash -c touch mydir/filea; echo touched number 1
bulksubmit: submit /bin/bash -c touch mydir/filea; echo touched number 2
bulksubmit: submit /bin/bash -c touch mydir/filea; echo touched number 3
bulksubmit: submit /bin/bash -c touch mydir/filea; echo touched number 4
bulksubmit: submit /bin/bash -c touch mydir/fileb; echo touched number 1
bulksubmit: submit /bin/bash -c touch mydir/fileb; echo touched number 2
bulksubmit: submit /bin/bash -c touch mydir/fileb; echo touched number 3
bulksubmit: submit /bin/bash -c touch mydir/fileb; echo touched number 4
bulksubmit: submit /bin/bash -c touch mydir/filec; echo touched number 1
bulksubmit: submit /bin/bash -c touch mydir/filec; echo touched number 2
bulksubmit: submit /bin/bash -c touch mydir/filec; echo touched number 3
bulksubmit: submit /bin/bash -c touch mydir/filec; echo touched number 4
bulksubmit: submit /bin/bash -c touch mydir/filed; echo touched number 1
bulksubmit: submit /bin/bash -c touch mydir/filed; echo touched number 2
bulksubmit: submit /bin/bash -c touch mydir/filed; echo touched number 3
bulksubmit: submit /bin/bash -c touch mydir/filed; echo touched number 4
```
Note that multiple inputs are combined, so in this example the same file gets touched 4 times, but with a different message for each based on the 2nd input. Maybe what you wanted was something that used the combinations like:
```console
$ flux bulksubmit --dry-run /bin/bash -c 'touch mydir/{0}/{1}; echo touched {0}/{1}' ::: dira dirb dirc dird ::: {1..4}
bulksubmit: submit /bin/bash -c touch mydir/dira/1; echo touched dira/1
bulksubmit: submit /bin/bash -c touch mydir/dira/2; echo touched dira/2
bulksubmit: submit /bin/bash -c touch mydir/dira/3; echo touched dira/3
bulksubmit: submit /bin/bash -c touch mydir/dira/4; echo touched dira/4
bulksubmit: submit /bin/bash -c touch mydir/dirb/1; echo touched dirb/1
bulksubmit: submit /bin/bash -c touch mydir/dirb/2; echo touched dirb/2
bulksubmit: submit /bin/bash -c touch mydir/dirb/3; echo touched dirb/3
bulksubmit: submit /bin/bash -c touch mydir/dirb/4; echo touched dirb/4
bulksubmit: submit /bin/bash -c touch mydir/dirc/1; echo touched dirc/1
bulksubmit: submit /bin/bash -c touch mydir/dirc/2; echo touched dirc/2
bulksubmit: submit /bin/bash -c touch mydir/dirc/3; echo touched dirc/3
bulksubmit: submit /bin/bash -c touch mydir/dirc/4; echo touched dirc/4
bulksubmit: submit /bin/bash -c touch mydir/dird/1; echo touched dird/1
bulksubmit: submit /bin/bash -c touch mydir/dird/2; echo touched dird/2
bulksubmit: submit /bin/bash -c touch mydir/dird/3; echo touched dird/3
bulksubmit: submit /bin/bash -c touch mydir/dird/4; echo touched dird/4
```